oxidation/libparsec/crates/client_types/src/lib.rs

```diff
@@ -1,11 +1,13 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
 
-mod error;
-mod local_device;
-mod local_device_file;
-mod local_manifest;
+pub mod error;
```

Comment:
we want simple import, so exposing sub modules is making things more complexe for no reason

we should be doing something like:
```rust
mod error;
mod local_device;
mod local_device_file;
mod local_manifest;

pub mod prelude {
    pub use super::error::*;
    pub use super::local_device::*;
    pub use super::local_device_file::*;
    pub use super::local_manifest::*;
}

pub use prelude::*;
```